InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -7372,9 +7372,9 @@ public int iwVerifyType(string str, int type)
 
         private Object AutoCastString(string str)
         {
-            if(str == "") return str;
+            if(String.IsNullOrEmpty(str)) return str;
```

Comment:
Not sure this is an operational semantic change, but even if it is, the result is the avoidance of a null exception on the next line.